packages/node-opcua-server-discovery/source/opcua_discovery_server.ts

```diff
@@ -55,9 +55,8 @@ function hasCapabilities(serverCapabilities: UAString[] | null, serverCapability
     return !!serverCapabilities.join(" ").match(serverCapabilityFilter);
 }
 
-export interface OPCUADiscoveryServerOptions extends OPCUABaseServerOptions {
+export interface OPCUADiscoveryServerOptions extends OPCUABaseServerOptions, OPCUAServerEndpointOptions {
```

Comment:
I would rather add a àlthername hostname property to the OPCUADiscoveryServerOptions  interface

OPCUAServerEndpointOptions  contains too many options that are not relevant to LDS server.